crates/libs/bindgen/src/rust/extensions/impl/Foundation/Collections/Iterable.rs

```diff
@@ -27,52 +28,54 @@ where
     T: windows_core::RuntimeType + 'static,
     T::Default: Clone,
 {
-    owner: IIterable<T>,
+    owner: windows_core::ComObject<StockIterable<T>>,
     current: std::sync::atomic::AtomicUsize,
 }
 
-impl<T> IIterator_Impl<T> for StockIterator<T>
+impl<T> IIterator_Impl<T> for StockIterator_Impl<T>
 where
     T: windows_core::RuntimeType,
     T::Default: Clone,
 {
     fn Current(&self) -> windows_core::Result<T> {
-        let owner: &StockIterable<T> = unsafe { windows_core::AsImpl::as_impl(&self.owner) };
+        let owner: &StockIterable<T> = &self.owner;
         let current = self.current.load(std::sync::atomic::Ordering::Relaxed);
 
-        if owner.values.len() > current {
+        if self.owner.values.len() > current {
             T::from_default(&owner.values[current])
         } else {
             Err(windows_core::Error::from(windows_core::imp::E_BOUNDS))
         }
     }
 
     fn HasCurrent(&self) -> windows_core::Result<bool> {
-        let owner: &StockIterable<T> = unsafe { windows_core::AsImpl::as_impl(&self.owner) };
+        let owner: &StockIterable<T> = &self.owner;
         let current = self.current.load(std::sync::atomic::Ordering::Relaxed);
 
         Ok(owner.values.len() > current)
     }
 
     fn MoveNext(&self) -> windows_core::Result<bool> {
-        let owner: &StockIterable<T> = unsafe { windows_core::AsImpl::as_impl(&self.owner) };
+        let owner: &StockIterable<T> = &self.owner;
         let current = self.current.load(std::sync::atomic::Ordering::Relaxed);
 
         if current < owner.values.len() {
-            self.current.fetch_add(1, std::sync::atomic::Ordering::Relaxed);
+            self.current
+                .fetch_add(1, std::sync::atomic::Ordering::Relaxed);
```

Comment:
Is this just formatting?

Reply:
Yes.  These files weren't being run through rustfmt before.